src/context/analysisResultStateSlice.ts

```diff
@@ -246,7 +246,7 @@ const createAPIRequestParams = (
 
   const { wcsConfig } = geotiffLayer;
   const dateValue =
-    !wcsConfig || wcsConfig.disableDateParam === false ? date : undefined;
+    !wcsConfig || !wcsConfig.disableDateParam ? date : undefined;
```

Comment:
Could we simplify this as simply:

`!wcsConfig?.disableDateParam` ? date : undefined

Reply:
included in [8b7d0aa](https://github.com/WFP-VAM/prism-app/pull/586/commits/8b7d0aa5bb44c1223048f3c19d17b3fb24785e86)